.github/workflows/land-blocking.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Land-blocking Test
+
+on:
+  push:
+    branches:
+      - auto
```

Comment:
@metajack I supposed this is the branch to block on when the cluster test fails?